packages/aws-cdk/lib/api/hotswap-deployments.ts

```diff
@@ -51,6 +58,10 @@ export async function tryHotswapDeployment(
     return undefined;
   }
 
+  if (hotswapLogMonitor) {
+    await findCloudWatchLogGroups(listStackResources, evaluateCfnTemplate, hotswapLogMonitor);
+  }
```

Comment:
I like this direction, but it's still only available if hotswapping is on - wouldn't work for `cdk watch --no-hotswap`, for instance, and I don't see any reason why it shouldn't.

We probably need to move this call to some higher-level in "watch" call chain than here.

Reply:
refactored to no longer be dependent on hotswap. Now as soon as you run `cdk watch` even if there are no changes, we will start monitoring the log groups.